src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -211,15 +211,13 @@ private void TerminateRunningProcess()
             {
                 if (Process == null || Process.HasExited)
                 {
-                    logger.LogDebug($"[eslint-bridge] [process id: {processId}] Cannot terminate eslint-bridge process: no process is running");
```

Comment:
TerminateRunningProcess is called when the class is Disposed, which happens whenever a solution is closed. 
If no JS/TS file was analyzed, this would still be printed. It's redundant so I've removed it.